### DISCUSSION TITLE
Has Sharing Changed?

### DISCUSSION THREAD

**Discussion by TheGallows** [jgraph/drawio#4455] [Q&A]
Hello! I have a drawing that I share to my org via Google Drive. Previously (I believe) those shared on it were able to click on the link and be brought to the drawing interface where they could pan around and zoom and all that fun vector format stuff. Now though, once they click the link they're faced with a couple of dialog boxes, culminating in viewing a PDF (that isn't dark mode :0 ) that has limited ability to Zoom and is too blurry. 

Yet when I, the owner of the drawing, click through the link, I still get the lovely editable version. How can I share this so that others in my org can interact (not necessarily edit) with it like I can again?

Thank you!

**Comment by davidjgraph** [comment]
I can't follow any of that, screenshots?

**Reply by TheGallows** [reply]
I can't get screenshots this minute but basically I need people in my organization to be able to view the draw.io file but instead all they can view is a pdf.

**Comment by TheGallows** [comment]
Hi David, is there a solution?

**Reply by davidjgraph** [reply]
I have no idea what the problem is yet.

**Comment by TheGallows** [comment]
I'll get you some screenshots today
